src/Modules/TeamSettingsModule/info.toml

```diff
@@ -0,0 +1,11 @@
+﻿[info]
+# A unique name for this module, this is used as a identifier
+name = "TeamSettings"
```

Comment:
The naming convention for modules is `<ModuleName>Module`, remember to change relevant references other places in the code when changing this.